inspec.yml

```diff
@@ -0,0 +1,9 @@
+name: linux-headers
+title: Habitat Core Plan linux-headers
+maintainer: smarshall@chef.io
```

Comment:
Should we put chef-core-planners@chef.io as the default email going forwards thinking about it?

Reply:
yep - we should

Reply:
Or humans@habitat.sh I guess, what do you think @davymcaleer ?

Reply:
Want me to change the grep / mysql inspec.yml files as well?

Reply:
hmm - chef-core-planners is this group - probably better as that, but humans is a wider overall group - I doubt it matters a whole lot, as long as it's consistent. We'll get issues raised rather than emails, bit ancient.
humans@habitat.io - keeps it inline with Core Plans